support/docker/dev/supervisord.conf

```diff
@@ -0,0 +1,3 @@
+[program:postgresql]
+user = user
+command = sudo -u postgres /usr/lib/postgresql/9.5/bin/postgres -D /var/lib/postgresql/9.5/main -c config_file=/etc/postgresql/9.5/main/postgresql.conf
```

Comment:
Nit: This command is very long. Why not make it shorter by using `user = postgres` like in the supervisor config example you found?

Reply:
I tried it the way you suggest, it didn't work, I don't have much time digging up why it didn't work, so meh 🤷‍. Happy to review a follow-up change to this if you're in the mood of making your PR thereafter.